test/e2e-framework/components/datadog/agent/kubernetes_helm.go

```diff
@@ -160,6 +164,8 @@ func NewHelmInstallation(e config.Env, args HelmInstallationArgs, opts ...pulumi
 
 	if args.GKEAutopilot {
 		values = buildLinuxHelmValuesAutopilot(baseName, agentImagePath, agentImageTag, clusterAgentImagePath, clusterAgentImageTag, randomClusterAgentToken.Result)
+	} else if args.OTelAgentGateway {
+		values = buildLinuxHelmValues(baseName, agentImagePath, agentImageTag, clusterAgentImagePath, clusterAgentImageTag, randomClusterAgentToken.Result, !args.DisableLogsContainerCollectAll, false, args.FIPS)
```

Comment:
What does `args.FIPS` mean ?

Reply:
This is an existing param to deploy with the FIPS agent image, it shouldn't be relevant to gateway tests but is needed for the function call